config/initializers/rolify.rb

```diff
@@ -1,7 +1 @@
-Rolify.configure do |config|
-  # By default ORM adapter is ActiveRecord. uncomment to use mongoid
-  # config.use_mongoid
-
-  # Dynamic shortcuts for User class (user.is_admin? like methods). Default is: false
-  config.use_dynamic_shortcuts
-end
+Rolify.configure(&:use_dynamic_shortcuts)
```

Comment:
-1 on this one, given there are other config values that could be set (that might get updated with new releases) and the comments are somewhat useful